src/http_api_handler.erl

```diff
@@ -89,21 +89,27 @@ handle_request(<<"POST">>, _, json, Req, State) ->
             {true, Req2, State}
     end;
 
+handle_request(<<"DELETE">>, <<"/api/v1/contexts">>, json, Req, State) ->
+    ok = ergw_api:delete_contexts(length(ergw_api:contexts(all))),
```

Comment:
It could be worth saving 
length(ergw_api:contexts(all))
for later use in this function.

Reply:
No, the values from both calls will no be identical.

Reply:
Yes, of course. My mistake.